src/main/java/org/openstreetmap/atlas/generator/PbfLocator.java

```diff
@@ -124,12 +137,9 @@ public PbfLocator(final PbfContext pbfContext, final Map<String, String> spark)
                 .flatMap(PbfLocator.this::tilesCoveringPartially);
         final Set<Shard> tileSet = Iterables.asSet(tileIterable);
         logger.trace("Found tiles {} for MultiPolygon {}", tileSet, multiPolygon.toSimpleString());
-        final Iterable<LocatedPbf> resourcesWithNulls = Iterables.stream(tileSet)
-                .filter(tile -> !innerCovers(tile, inners)).map(shard -> (SlippyTile) shard)
-                .map(this.tileToLocatedPbf);
-        // Filter out all the null resources, meaning where the PBFs were not found. (Ocean for
```

Comment:
Why did the comment disappear? We are still filtering and it added some context as to why.

Reply:
Makes sense!

Reply:
Oh I see. I could still use that comment above... Let me do that!